components/provisioner/pkg/gqlschema/schema.graphql

```diff
@@ -44,10 +44,8 @@ type AzureProviderConfig {
 }
 
 type AWSProviderConfig {
-    zone: String
+    zones: [AWSZone]!
```

Comment:
possible renaming needed here:
```
"errors": [
      {
        "message": "Fields \"zones\" conflict because they return conflicting types [String!]! and [AWSZone]!. Use different aliases on the fields to fetch both if this was intentional.",
        ```